CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ project(DALI CUDA CXX C)
 set(DALI_ROOT ${PROJECT_SOURCE_DIR})
 set(CUDA_VERSION "${CMAKE_CUDA_COMPILER_VERSION}")
 
+include(${DALI_ROOT}/cmake/Utils.cmake)
```

Comment:
That's already here, 40 lines below. It'd keep the one with relative path
```
include(cmake/Utils.cmake)
```

Reply:
will do